src/components/kpiWidget/kpiWidget.js

```diff
@@ -53,7 +53,7 @@ class KpiWidget extends Component {
   scheduleAutoUpdate() {
     this.intervalId = setTimeout(() => {
       this.props.intervalChanged(this.state.selectedValue);
-    }, Config.msTelemetryApiRefreshInterval * 1000);
+    }, Config.INTERVALS.KPI_REFRESH * 1000);
```

Comment:
The interval configs should be consistent. The other values are already in ms so convert `KPI_REFRESH` to ms as well, removing the `* 1000` here.